db/migrate/20220525152551_remove_unique_sub_index_from_tombstones.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniqueSubIndexFromTombstones < ActiveRecord::Migration[7.0]
+  def change
+    remove_index :tombstones, column: :sub
+    add_index :tombstones, :sub, if_not_exists: true
```

Comment:
huh, cool!
So that's a projection from any potential clash during migration?

https://blog.saeloun.com/2020/05/11/if-not-exists-for-index.html

Not come across that before

Reply:
Yeah exactly - we've got a lot of migrations going around at the moment so I thought it was safest